src/main/java/org/team1540/bobafett/RobotContainer.java

```diff
@@ -44,17 +46,25 @@ public RobotContainer() {
      * edu.wpi.first.wpilibj2.command.button.JoystickButton}.
      */
     private void configureButtonBindings() {
-        new JoystickButton(controller, XboxController.Button.kX.value)
-                .whenPressed(new PigeonTurn(drivetrain, 90));
-        new JoystickButton(controller, XboxController.Button.kB.value)
+        new JoystickButton(pilotController, XboxController.Button.kX.value)
+                .whileActiveContinuous(new PigeonTurn(drivetrain, 90));
+
+        new JoystickButton(pilotController, XboxController.Button.kLeftStick.value)
```

Comment:
I would caution against using the stick buttons, they're easy to press accidentally